CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -4,8 +4,14 @@
 
 ## Changes by Kind
 
+### Chore
+
+- KubeOne is now built with Go 1.23.4 ([#3513](https://github.com/kubermatic/kubeone/pull/3513), [@kubermatic-bot](https://github.com/kubermatic-bot))
```

Comment:
We usually don't leave the Chore section, instead we manually edit the changelog:

- If it's an update, we create the `Updates` section. This concrete change could go into `Updates` and `Other`, see an example here: https://github.com/kubermatic/kubeone/blob/main/CHANGELOG/CHANGELOG-1.9.md#others
- If it's not an update, we put it in the `Other (Cleanup or Flake)` section

Someday, we could see what it takes to create a new kind such as `update`, so we can do something like `/kind update` and have the changelog generator pick it up correctly out of the box.